docs/src/pages/system/the-sx-prop/the-sx-prop.md

```diff
@@ -0,0 +1,147 @@
+# The `sx` prop
+
+<p class="description">The `sx` prop is a shortcut for defining custom style overrides for the components, using the theme.</p>
+The property is a superset of CSS that packages [all the style functions](/system/basics/#all-inclusive) that are exposed in `@material-ui/system`.
+You can specify any valid CSS using this prop.
+
+## Example
+
+{{"demo": "pages/system/the-sx-prop/Example.js", "bg": true, "defaultCodeOpen": false}}
+
+On the example above, you can notice that some of the values are not valid CSS properties.
+This is because, the CSS properties are mapped to specific paths in the theme.
+In the following sections, you will learn how different CSS properties are mapped to specigic parts of the theme.
+
+## Borders
+
+The `border` property can receive only a number as a value and and it will create a solid black border using the number as the width.
+
+```jsx
+<Box sx={{ border: 1 }} /> // eqivalent as border: '1px solid black'
+```
+
+The `borderColor` property can receive a string, which represents the path in the `theme.palette`.
+
+```jsx
+<Box sx={{ borderColor: 'primary.main' }} /> // eqivalent as borderColor: theme => theme.palette.primary.main
+```
+
+The `borderRadius` properties multiples the value it receives by the `theme.shape.borderRadius` value (the default for the value is `4px`).
+
+```jsx
+<Box sx={{ borderRadius: 2 }} /> // eqivalent as borderRadius: theme => 2 * theme.shape.borderRadius
+```
+
+## Display
+
+The `displayPrint` property allows you to specify CSS `display` value, that will be applied only for priting.
+
+```jsx
+<Box sx={{ displayPrint: 'none' }} /> // eqivalent as '@media print': { display: 'none' }
+```
+
+## Grid
+
+The grid CSS properties `gap`, `rowGap` and `columnGap` multiply the values they receive by the `theme.spacing` value (the default for the value is `8px`).
+
+```jsx
+<Box sx={{ gap: 2 }} /> // eqivalent as gap: theme => theme.spacing(2)
+```
+
+## Palette
+
+The `color` and `backgroundColor` properties can receive a string, which represents the path in the `theme.palette`.
+
+```jsx
+<Box sx={{ color: 'primary.main' }} /> // eqivalent as color: theme => theme.palette.primary.main
+```
+
+The `backgroundColor` property is also available trough its alias `bgcolor`.
+
+```jsx
+<Box sx={{ bgcolor: 'primary.main' }} /> // eqivalent as backgroundColor: theme => theme.palette.primary.main
+```
+
+## Positions
+
+The `zIndex` property maps its value to the `theme.zIndex` value.
+
+```jsx
+<Box sx={{ zIndex: 'tooltip' }} /> // eqivalent as backgroundColor: theme => theme.zIndex.tooltip
+```
+
+## Shadows
+
+The `boxShadow` property maps its value to the `theme.shadows` value.
+
+```jsx
+<Box sx={{ boxShadow: 1 }} /> // eqivalent as boxShadow: theme => theme.shadows[1]
+```
+
+## Sizing
+
+The sizing properties: `width`, `height`, `minHeight`, `maxHeight`, `minWidth` and `maxWidth` are using the following custom transform function for the value:
```

Comment:
This, in my opinion can be super confusing (the implementation, not the docs). IMO we would be better off allowing explicitly setting "25%" instead of 0.25.

Reply:
We already accept percent values (and any other regular CSS value). The values between 0 and 1 are complementary, so I don't think it is that big a deal, but I see your point :) @oliviertassinari initially implemented this logic I think, so he may have more input on it.

Reply:
Honestly, I don't like the fraction conversion to percentage at all. It is counter intuitive with normal css. Anyway, I don't expect us to change it right now.